tabletop/core/game.js

```diff
@@ -161,6 +166,15 @@ Game.prototype.nextPlayer = function() {
 };
 
 /**
+ * Returns the next player, but does not switch
+ * Override to provide more logic on determining the next player
+ * @returns {void}
```

Comment:
yeah i gotta go through and add this JSdoc stuff for the other methods too. good catch here